tests/e2e/yamcs/telemetryTables.e2e.spec.mjs

```diff
@@ -77,4 +78,58 @@ test.describe("Telemetry Tables tests @yamcs", () => {
         await expect(page.getByRole('button', { name: 'SHOW LIMITED' })).toBeVisible();
     });
 
+    test('Telemetry tables are sorted in desc order correctly', async ({ page }) => {
+        await setRealTimeMode(page);
+
+        //navigate to CCSDS_Packet_Length with a specified realtime window
+        await page.goto('./#/browse/taxonomy:spacecraft/taxonomy:~myproject/taxonomy:~myproject~CCSDS_Packet_Length?tc.mode=local&tc.startDelta=5000&tc.endDelta=5000&tc.timeSystem=utc&view=table', {waitUntil: 'domcontentloaded'});
+
+        // wait 5 seconds for the table to fill
+        await page.waitForTimeout(5*1000);
+        // pause the table
+        await page.getByLabel('Pause').click();
+        const telemTableDesc = await page.getByLabel("CCSDS_Packet_Length table content");
+
+        // assert that they're in desc order
+        expect(await assertTableRowsInOrder(telemTableDesc, 'desc')).toBe(true);
+
+        // Unpause
+        await page.getByLabel('Play').click();
+
+        // flip sort order
+        await page.locator('thead div').filter({ hasText: 'Timestamp' }).click();
+
+        // wait for x seconds
```

Comment:
when setting time const delays like this, we should turn it into a variable. We should also be failing our linting checks here...